honeycombio/raas-hnytest
# Update EU1 workflow to use same refinery-values file as US1 workflow

## Short description of the changes

This uses the `--set` option for the helm deploy to set the API URL when deploying to EU1, so that when we deploy, we can just use one values file for both environments, rather than two.  Since the URL and the ACM ARN are referenced this way as well, there's really no need to have multiple values files for each ecosystem.

## How to verify that this has the expected result

Once merged, try to run the EU1 refinery deployment workflow to verify functionality.
